controllers/v1/bulkUploadRequest.js

```diff
@@ -5,159 +5,369 @@
  * Description : bulk upload operations 
  */
 
-const bulkUploadHelper = require(MODULES_BASE_PATH + "/bulk-upload-request/helper.js");
- 
+const bulkUploadHelper = require(MODULES_BASE_PATH + "/bulkUploadRequest/helper.js");
+
+/**
+   * BulkUploadRequest
+   * @class
+*/
+module.exports = class BulkUploadRequest extends Abstract {
+  constructor() {
+    super(schemas["bulkUploadRequest"]);
+  }
+
+  static get name() {
+    return "bulkUploadRequest";
+  }
+
   /**
-     * Forms
-     * @class
- */
-module.exports = class PlatformRolesExt extends Abstract {
-    constructor() {
-      super(schemas["bulkUploadRequest"]);
-    }
-  
-    static get name() {
-      return "bulkUploadRequest";
-    }
-
-    /**
-     * @api {get} /admin-service/api/v1/bulkUploadRequest/bulkUserUpload 
-     * Upload bulk user Upload
-     * @apiVersion 1.0.0
-     * @apiGroup Bulk Upload
-     * @apiHeader {String} X-authenticated-user-token Authenticity token
-     * @apiSampleRequest /admin-service/api/v1/bulkUploadRequest/bulkUserUpload
-     * @apiUse successBody
-     * @apiUse errorBodyuser
-    */
+  * @api {get} /admin-service/api/v1/bulkUploadRequest/bulkUpload 
+  * For bulk upload
+  * @apiVersion 1.0.0
+  * @apiGroup Bulk Upload
+  * @apiHeader {String} X-authenticated-user-token Authenticity token
+  * @apiSampleRequest /admin-service/api/v1/bulkUploadRequest/bulkUpload
+  * @apiParam {File} uploadFile users list file of type CSV. 
+  * @apiUse successBody
+  * @apiUse errorBody
+  * @apiParamExample {json} Response:
+  * {
+  *  result: { requestId: "ADSASSq31" },
+  *  message:"Request Submitted Successfully"
+  * }
+  **/
 
   /**
-   * bulk User Upload
+   * For bulk upload
    * @method
-   * @name bulkUserUpload
+   * @name bulkUpload
    * @param  {req}  - requested data.
-   * @returns {json} Response consists of created user details
+   * @returns {json} Response consists of request details
   */
 
- bulkUserUpload(req) {
-  return new Promise(async (resolve, reject) => {
-    try {
-
-
-      // console.log("req.files",req.files);
-      if (!req.files || !req.files.userCreationFile) {
-        throw { 
-            status: httpStatusCode["bad_request"].status, 
-            message: httpStatusCode["bad_request"].message 
-        };
-      } 
-
-      let uploadRequest = await bulkUploadHelper.bulkUserUpload(req,req.userDetails.userId);
-      
-      
-
-      return resolve(uploadRequest);
-    } catch (error) {
-
-      return reject({
-        status:
-          error.status ||
-          httpStatusCode["internal_server_error"].status,
-
-        message:
-          error.message ||
-          httpStatusCode["internal_server_error"].message
-      });
-    }
-  });
-}
+  bulkUpload(req) {
```

Comment:
@rakeshSgr - Make the endpoint to create

Reply:
@rakeshSgr - Explore middleware flow, where each module has its own bulk upload. Middleware generates the request if and forwards the request to the controller